clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -279,7 +279,6 @@ public DaVinciBackend(
           null);
 
       ingestionService.start();
-      ingestionService.addIngestionNotifier(ingestionListener);
```

Comment:
Why remove this?

Reply:
This is the "bug" I found during the debugging - DVC actually register the same ingestion notifier twice - the 2nd invocation is in bootstrap() method. So today every status will report twice. I found this during testing and hence remove one of it.